rust/platform/triple_mappings.bzl

```diff
@@ -196,58 +198,16 @@ def abi_to_constraints(abi):
         # figure out how they're doing this
         return []
 
-def triple_to_system(triple):
+def triple_to_system(target_triple):
     """Returns a system name for a given platform triple
 
     Args:
-        triple (str): A platform triple. eg: `x86_64-unknown-linux-gnu`
+        target_triple (str): A platform triple. eg: `x86_64-unknown-linux-gnu`
 
     Returns:
         str: A system name
     """
-    if triple == "wasm32-wasi":
-        return "wasi"
-
-    component_parts = triple.split("-")
-    if len(component_parts) < 3:
-        fail("Expected target triple to contain at least three sections separated by '-'")
-
-    return component_parts[2]
-
-def triple_to_arch(triple):
```

Comment:
Since this module is public, these are part of our public API. Can you restore the functions and maybe print a warning that `@rules_rust//rust/platforms/triple.bzl` should be used instead?

Reply:
Done.